scripts/pretest.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# interrupt on any error
+set -e
+
+# load .env file to get NODE_ENV
+if [ -f ".env" ]; then
+  source ".env"
+fi
+
```

Comment:
you don't need this. `pretest` will only be run in development and circleci environments.